bool getValues
---
bool ledValB
uint16 servoVal